style.css

```diff
@@ -26,6 +30,121 @@ header {
   background: #3c3a39;
 }
 
+.popup-menu {
```

Comment:
- [ ] Your design should match to the original design. Kindly compare your design to the original.


Your image            |  Original image
:-------------------------:|:-------------------------:
<img width="1440" alt="Screenshot 2023-01-03 at 18 29 21" src="https://user-images.githubusercontent.com/69011963/216935045-72f68345-70b3-43fc-a7ca-c0539a3cc0d2.png"> | <img src="https://user-images.githubusercontent.com/69011963/216935063-614337d1-296d-4762-b04a-7fd439d80940.png" width="1440"/>

Your mobile menu should cover the page.